eventorganizer.api/src/main/java/com/event/organizer/api/controller/AccountController.java

```diff
@@ -48,4 +46,9 @@ public AppUser changeAccountRole(@RequestBody AccountRolesRequestDto accountRole
     public AppUser changeAccountStatus(@RequestBody AccountStatusRequestDto accountStatusRequestDto, Principal user) {
         return userService.changeAccountStatus(user.getName(), accountStatusRequestDto);
     }
+
+    @PutMapping("/addProfilePicture")
+    public AppUser uploadProfilePicture(@RequestBody ProfilePictureRequestDto profilePictureRequestDto, Principal user) {
+        return userService.uploadProfilePicture(user.getName(), profilePictureRequestDto.getUrl());
```

Comment:
put the Princpal parameter as a second one in the service as it is in the controller
 
`        return userService.uploadProfilePicture(profilePictureRequestDto.getUrl(), user.getName());